Mathlib/LinearAlgebra/FreeModule/PID.lean

```diff
@@ -624,7 +624,7 @@ theorem Ideal.smithCoeffs_ne_zero (b : Basis ι R S) (I : Ideal S) (hI : I ≠ 
 #align ideal.smith_coeffs_ne_zero Ideal.smithCoeffs_ne_zero
 
 instance (F : Type u) [CommRing F] [Algebra F R] (b : Basis ι R S) {I : Ideal S} (hI : I ≠ ⊥) (i) :
-    Module F (R ⧸ Ideal.span ({I.smithCoeffs b hI i} : Set R)) := by
+    Module F (R ⧸ Ideal.span ({I.smithCoeffs b hI i} : Set R)) :=
   Quotient.module' _
```

Comment:
I wasn't suggesting that you _actually_ replace this with `Quotient.module'`; just that you check the comment was accurate (it previously wasn't).

I don't really understand why we added this instance in Lean 3 in the first place though if `apply_instance` could find it there.

Reply:
I wasn't able to do anything locally so I thought I'd let CI tell me

Reply:
I guess since #4821 works just fine without this, I'll delete this PR